README.md

```diff
@@ -1,17 +1,121 @@
+
+```shell
+$ oc describe --namespace=openshift-ingress-operator ingresscontroller/default
+```
+
+## Manage
+
+Ingress Operator provides the OpenShift [ingresscontroller API](). Create and
+edit `ingresscontroller.operator.openshift.io` resources to manage ingress
+controllers.
+
+Interact with ingress controllers using the `oc` command. Every ingress
+controller lives in the `openshift-ingress-operator` namespace.
+
+To scale an ingress controller:
+
+```shell
+$ oc scale --replicas=1 --namespace=openshift-ingress-operator ingresscontroller/<name>
+```
+
+## Customize
+
+Create new `ingresscontroller` resources in the `openshift-ingress-operator`
+namespace.
+
+To edit an existing ingress controller:
+
+```shell
+$ oc edit --namespace=openshift-ingress-operator ingresscontroller/<name>
+```
+
+**Important:** Updating an ingress controller may lead to disruption for public
+facing network connections as a new ingress controller revision may be rolled
+out.
+
+Refer to the [ingresscontroller API]() for full details on defaults and
+customizing an ingress controller. The most important initial customizations are
+domain and endpoint publishing strategy, as they *cannot currently be changed
+after the ingress controller is created*.
+
+### Endpoint publishing
+
+The `.spec.endpointPublishingStrategy` field is used to publish the ingress
+controller endpoints to other networks, enable load balancer integrations, etc.
+
+Every strategy is described in detail in the [ingresscontroller API](). A brief
+design diagram for each is shown below.
 
+#### LoadBalancerService
 
-## Reporting issues
+The [LoadBalancerService]() strategy publishes an ingress controller using a
+Kubernetes [LoadBalancer
+Service](https://kubernetes.io/docs/concepts/services-networking/#loadbalancer)
+and on some platforms offers managed wildcard DNS.
 
-Bugs are tracked in [Bugzilla](https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform&version=4.0.0&component=Routing).
+![Image of LoadBalancerService](docs/images/endpoint-publishing-loadbalancerservice.png)
+
+#### HostNetwork
+
+The [HostNetwork]() strategy uses host networking to publish the ingress
+controller on node ports where the ingress controller is deployed.
+
+![Image of HostNetwork](docs/images/endpoint-publishing-hostnetwork.png)
+
+#### Private
+
+The [Private]() strategy does not publish the ingress controller.
+
+![Image of Private](docs/images/endpoint-publishing-private.png)
+
+## Troubleshoot
+
+Use the `oc` command to troubleshoot operator issues.
+
+To inspect the operator's status:
+
+```shell
+$ oc describe clusteroperators/ingress
+```
+
+To view the operator's logs:
+
+```shell
+$ oc logs --namespace=openshift-ingress-operator deployments/ingress-operator
+```
+
+To inspect the status of a particular ingress controller:
+
+```shell
+$ oc describe --namespace=openshift-ingress-operator ingresscontroller/<name>
+```
```

Comment:
How about logs for the ingress controller (i.e., router)?
```shell
$ oc logs --namespace=openshift-ingress deployments/router-<name>
```
(Are we going to rename "router" to "ingress-controller"?)

Reply:
Waffled on this one... is exposing a potentially volatile naming convention problematic? Is there some other way we could direct users to the logs via the ingresscontroller?  Random ideas:

* Implement `oc logs` for an `ingresscontroller` to abstract discovery of the deployment
* Report the deployment name on status
* Document a stable selector for the deployment to pass to `oc logs --selector`